Player.h

```diff
@@ -0,0 +1,27 @@
+#ifndef PLAYER_H
+#define PLAYER_H
+
+#include <array>
+#include "Card.h"
+#include "Deck.h"
+#include "g_constants.h"
+
+using PlayerHand = std::array<Card, g_constants::g_maximumHand>;
+
+class Player
+{
+private:
+    int m_score{};
+    PlayerHand m_hand{};
+    
+public:
+    Player() = default;
```

Comment:
Since you're not manually writing any other constructors, you don't need to write this out at all. I think this is only useful if you wanted to write your own version of one of the other implicitly generated constructors, but you wanted this one to use the default.